src/AltinnCore/Common/Models/Data.cs

```diff
@@ -15,6 +15,12 @@ public class Data
         [JsonProperty(PropertyName = "id")]
         public string Id { get; set; }
 
+        /// <summary>
```

Comment:
duplication of Data and Instance classes should be avoided. Consider reference to Storage.Interface

Reply:
I think i will wait until we decide on publishing private nugets or public. We can discuss it on Monday